src/Categories/Morphism/Properties.agda

```diff
@@ -150,3 +150,66 @@ _∘↠_ : B ↠ C → A ↠ B → A ↠ C
 f ∘↠ g = record { mor = mor f ∘ mor g ; epi = Epi-∘ (epi f) (epi g) }
   where
     open _↠_
+
+--------------------------------------------------------------------------------
+-- Isomorphism from a section and a retraction
+
+EpicRetract⇒Iso : ∀ {X Y} {f : Y ⇒ X} {r : X ⇒ Y} → 
+                   r RetractOf f → Epi f → Iso f r
+EpicRetract⇒Iso {X} {Y} {f} {r} rf≈id e = record { 
+  isoˡ = rf≈id ; 
+  isoʳ = e (f ∘ r) id (begin
+    (f ∘ r) ∘ f ≈⟨ assoc ⟩
```

Comment:
Look at the various push/pull combinators - these two lines can be combined into one.

Reply:
I used those combinators and the ones suggested below, to make all proofs shorter